README.md

```diff
@@ -114,6 +114,8 @@ Elyra can be installed from PyPI:
   conda install -c conda-forge "elyra[all]>=3.7.0"
   ```
 
+  Install Elyra from PyPI ( Elyra >= 3.1.0 ):
+
   ```bash
   pip3 install --upgrade "elyra[all]>=3.1.0" && jupyter lab build
   ```
```

Comment:
I'm finding this section of the README confusing and inconsistent.  In some places, the version in the "header" matches that in the command, and in others not.  In some places, `&& jupyter lab build` is appended, and in others not.

I'm assuming the lab build phrase is no longer necessary due to "prebuilt extensions", but it seems like some explanation to that effect is necessary.  Or remove all of the `&& jupyter lab build` and add a section titled: "When is a lab build necessary" or something related to Lab builds with an explanation as to why it's not required for elyra >= 3.7.

I suspect the version number in the "header" on L112 needs to reflect `3.7.0` as well.

I'm also not understanding the significance of `3.1.0` but do see the significance of `< 2`, although I think the "and above" comment in "JupyterLab 2.x is supported on Elyra 1.0.0 and above" should be updated to be "and below 3.0.0".

I know these aren't all necessarily related to prebuilt extensions, but it seems like a good time to clean things up.

Reply:
L112 is an error, I'll fix that. I'll also add a line explaining why `jupyter lab build` is needed for install prior to 3.7

As for the 3.1 difference, that was when we added the `[all]` to the install line.